recipes/libbigwig/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import collect_libs, get, copy
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class LibBigWigConan(ConanFile):
+    name = "libbigwig"
+    description = "A C library for handling bigWig files"
+    topics = ("bioinformatics", "bigwig", "bigbed")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dpryan79/libBigWig"
+    license = "MIT"
+    settings = "arch", "build_type", "compiler", "os"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_curl": [True, False],
+        "with_zlibng": [True, False]
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_curl": True,
+        "with_zlibng": False
+    }
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        try:
+            del self.settings.compiler.libcxx
+        except Exception:
+            pass
+        try:
+            del self.settings.compiler.cppstd
+        except Exception:
+            pass
+
+        if self.options.with_curl:
+            self.options["libcurl"].with_imap = False
```

Comment:
Need to check if libcurl is available by default using these options, otherwise, won't be possible to provide pre-built packages for this recipe.